docs/openeuropa-components.md

```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
-    </tr>
-    <tr>
-        <td>
-            <a title="Repository" href="https://github.com/openeuropa/oe_webtools"><b>OpenEuropa Webtools</b></a><br/>
-            Drupal module providing webtools analytics integration such as Matomo (formerly PIWIK).
-        </td>
-        <td width="250">
-            <a title="Version" href="https://packagist.org/packages/openeuropa/oe_webtools"><img src="https://img.shields.io/packagist/v/openeuropa/oe_webtools.svg"/></a>
-            <a title="Build" href="https://drone.fpfis.eu/openeuropa/oe_webtools"><img src="https://drone.fpfis.eu/api/badges/openeuropa/oe_webtools/status.svg"/></a>
-            <a title="Downloads" href="https://packagist.org/packages/openeuropa/oe_webtools"><img src="https://img.shields.io/packagist/dt/openeuropa/oe_webtools.svg"/></a>
-        </td>
- <td><a href="https://github.com/imanoleguskiza">Imanol Eguskiza</a><br/>
-            <a href="https://github.com/upchuk">Daniel Sipos</a></td>
-    </tr>
-</table>
+Components are divided by track, each track has a few OpenEuropa team members responsible for it.
+
+### Editorial track
+
+Maintainers: [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Diogo](https://github.com/dxvargas)
+  
+Components:
+- [OpenEuropa Editorial](https://github.com/openeuropa/oe_editorial)
+
+### Multilingual track
+
+Maintainers: [Daniel](https://github.com/upchuk), [Diogo](https://github.com/dxvargas), [Imanol](https://github.com/imanoleguskiza)
+
+Components:
+- [OpenEuropa Multilingual](https://github.com/openeuropa/oe_multilingual)
+- [OpenEuropa Translation](https://github.com/openeuropa/oe_translation)
+
+### Webtools track
+
+Maintainers: [Francesco](https://github.com/brummbar), [Imanol](https://github.com/imanoleguskiza), [Sergii](https://github.com/sergepavle)
+
+Components:
+- [OpenEuropa Webtools](https://github.com/openeuropa/oe_webtools)
+
+### Theme track
+
+Maintainers: [Antonio](https://github.com/ademarco), [Imanol](https://github.com/imanoleguskiza), [Victor](https://github.com/voidtek)
+
+Components:
+- [OpenEuropa Theme](https://github.com/openeuropa/oe_theme)
+- [OpenEuropa ECL Twig Loader](https://github.com/openeuropa/ecl-twig-loader)
+
+### Content track
+
+Maintainers: [Francesco](https://github.com/brummbar), [Adam](https://github.com/nagyad), [Sergii](https://github.com/sergepavle), [Daniel](https://github.com/upchuk)
+
+Components:
+- [OpenEuropa Content](https://github.com/openeuropa/oe_content)
+- [OpenEuropa Paragraphs](https://github.com/openeuropa/oe_paragraphs)
+- [OpenEuropa Corporate Blocks](https://github.com/openeuropa/oe_corporate_blocks)
+- [OpenEuropa RDF SKOS](https://github.com/openeuropa/rdf_skos)
+
+### Media track
+
+Maintainers: [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Daniel](https://github.com/upchuk)
+
+Components:
+- [OpenEuropa Media](https://github.com/openeuropa/oe_media)
+- [OpenEuropa Oembed](https://github.com/openeuropa/oe_oembed)
```

Comment:
`media_avportal` is missing

Reply:
That's a dependency of oe_media, that's why it's not explicitly listed here